ui/components/app/wallet-overview/btc-overview.test.tsx

```diff
@@ -40,43 +41,63 @@ const mockNonEvmAccount = {
   type: BtcAccountType.P2wpkh,
 };
 
-function getStore(state?: Record<string, unknown>) {
-  return configureMockStore([thunk])({
-    metamask: {
-      ...mockState.metamask,
-      internalAccounts: {
-        accounts: {
-          [mockNonEvmAccount.id]: mockNonEvmAccount,
-        },
-        selectedAccount: mockNonEvmAccount.id,
-      },
-      // (Multichain) BalancesController
-      balances: {
-        [mockNonEvmAccount.id]: {
-          [MultichainNativeAssets.BITCOIN]: {
-            amount: mockNonEvmBalance,
-            unit: 'BTC',
-          },
-        },
-      },
-      // (Multichain) RatesController
-      fiatCurrency: 'usd',
-      rates: {
-        [Cryptocurrency.Btc]: {
-          conversionRate: '1.000',
-          conversionDate: 0,
-        },
+const mockBtcChain = {
+  active: true,
+  chainId: 'bip122:000000000019d6689c085ae165831e93',
```

Comment:
Maybe use the constant here, unless this was intended?

```suggestion
  chainId: MultichainNetworks.BITCOIN,
```